### ISSUE TITLE
UndefinedMethod for phpdoc fluent interface

### ISSUE THREAD

**Issue by vudaltsov** [vimeo/psalm#1196]
Hi!

```php
/**
 * @method $this setOrderItemId(int $orderItemId)
 * @method $this setPayerRunetId(int $payerRunetId)
 */
final class PayDeleteEndpoint extends AbstractDeleteEndpoint
{
    /**
     * {@inheritdoc}
     */
    protected function getEndpoint()
    {
        return '/pay/delete';
    }
}
```

```php
// ParticipationHandler
$client
    ->payDelete()
    ->setOrderItemId($item->Id)
    ->getRawResult();
```

ERROR: UndefinedMethod - src/Basket/Handler/ParticipationHandler.php:45:19 - Method App\Basket\Handler\ParticipationHandler::getrawresult does not exist ->getRawResult();

**Comment by muglug** [comment]
Hey!

Do you have `usePhpDocMethodsWithoutMagicCall="true"` in psalm.xml?

**Comment by vudaltsov** [comment]
@muglug, I tried it — didn't help. Here's a reproducer: https://github.com/vudaltsov/psalm-runet-id-magic-calls.

**Comment by vudaltsov** [comment]
```
git clone git@github.com:vudaltsov/psalm-runet-id-magic-calls.git
cd psalm-runet-id-magic-calls
composer install
vendor/bin/psalm
```

**Comment by muglug** [comment]
Thanks @vudaltsov - this was fixed with the fix to #1257
